pkgs/stderred/default.nix

```diff
@@ -0,0 +1,17 @@
+{ rustPlatform, stderred, makeWrapper }:
+
+rustPlatform.buildRustPackage {
+  pname = "stderred";
+  version = "0.1.0";
+
+  src = ./.;
```

Comment:
https://nix.dev/anti-patterns/language#reproducibility-referencing-top-level-directory-with 

While this isn't a top-level directory, I would still avoid using this pattern in case someone were to copy it, or if someone wanted to move stderred into its own repository.

Reply:
TIL!